src/services/formatService.ts

```diff
@@ -0,0 +1,167 @@
+            biometricPermit: "UK biometric residence permit (BRP) Welsh",
+            biometricCard: "UK biometric residence card (BRC) Welsh",
+            frontierPermit: "UK Frontier Worker permit Welsh",
+            passportIrishCard: "Passport or Irish passport card welsh",
+            ukBRP: "UK biometric residence permit (BRP) welsh",
+            ukBRC: "UK biometric residence card (BRC) welsh",
+            passCard: "UK accredited PASS card welsh",
+            ukEuDigitalCard: "UK or EU driver digital tachograph card welsh",
+            fullDrivingLicense: "UK, Channel Islands, Isle of Man and EU photocard driving licence (full or provisional) welsh",
+            ukForceCard: "UK HM Forces ID Card welsh",
+            ukArmedForceCard: "UK HM Armed Forces Veteran Card welsh",
+            ukFrontierPermit: "UK Frontier Worker permit welsh",
+            photoWorkPermit: "Photographic work permit (government issued) welsh",
+            photoimmigrationDoc: "Photographic immigration document welsh",
+            photoVisa: "Photographic Visa welsh",
+            ukFirearmsLicence: "UK, Channel Islands and Isle of Man Firearms Licence welsh",
+            photoIdPrado: "Photographic ID listed on PRADO welsh",
+            photoIdPradoHint: "Such as a National Identity Card (Pakistan), crew member certificate (South Africa), or Permanent Resident Card (USA). welsh",
+            birthCert: "Birth or adoption certificate welsh",
+            marriageCert: "Marriage or civil partnership certificate welsh",
+            noPhotoimmigrationDoc: "Non-photographic immigration document welsh",
+            noPhotoVisa: "Non-photographic visa welsh",
+            noPhotoWorkPermit: "Non-photographic work permit welsh",
+            bankStatement: "Bank or building society statement welsh",
+            rentalAgreement: "UK local authority or social housing rental agreement (for the person’s current address) welsh",
+            morgageStatement: "Mortgage statement (for the person’s current address) welsh",
+            taxStatement: "UK council tax statement (for the person’s current address welsh",
+            utilityBill: "Utility bill (for the person’s current address) welsh"
+        };
+
+        const documentMapping = lang === "en" ? documentMappingEn : documentMappingWelsh;
+
+        const formattedDocuments = documents.map(doc => {
+            const docText = documentMapping[doc] || doc;
+            return `• ${docText}`;
+        });
+
+        return formattedDocuments.join("<br>");
+    }
+
+    public static formatHowIdentityDocsChecked (option: string | undefined, lang: string): string {
+        if (!option) {
+            return "";
+        }
+        let result: string;
+
+        switch (option) {
+        case "OPTION1":
+            if (lang === "en") {
+                result = "Option 1 - The identity documents were checked using identity document validation technology (IDVT).";
```

Comment:
Why is it hard coded value here? Why not picking it up from json file?

Reply:
We can not pickup json within service also we have a different text which is on there in json.